tools/xabuild/xabuild.csproj

```diff
@@ -70,8 +47,13 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Runtime" Version="16.6.0" />
```

Comment:
I'm not sure using this `<PackageReference/>` will work? `xabuild` is this terrible thing that references `MSBuild.exe` directly.

Reply:
Commit reverted

Reply:
learnt it hard way.